packages/react/src/box/styles.ts

```diff
@@ -160,6 +190,9 @@ function isEntry(a: unknown): a is {
 	return !!a; // ie. not null or undefined
 }
 type LayoutProps = Partial<{
+	/** Sets the CSS display property.
```

Comment:
I think this is probably fine but it would be nice to just lean on `csstype`. E.g. if we do something like this:

```typescript
import { CSSObject } from '@emotion/react';

type Display = CSSObject['display'];
```

We get this:

![image](https://github.com/agriculturegovau/agds-next/assets/43688949/521d8c93-7397-403c-bd6f-e77c57bb3af9)

Reply:
Wrapping `ResponsiveProp` around it loses a bit of the intellisense shine, mind you.

Reply:
I think having a consistent format  for all of our ones which accept tokens or do a subset of the property is more beneficial personally